pod.go

```diff
@@ -833,6 +854,20 @@ func (p *Pod) setPodState(state State) error {
 	return nil
 }
 
+func (p *Pod) getContainer(containerID string) (*Container, error) {
+	if containerID == "" {
+		return &Container{}, errNeedContainerID
+	}
+
+	for _, c := range p.containers {
+		if c.ID() == containerID {
```

Comment:
You don't need the getter here because we are in the virtcontainers lib itself, you can do `c.id` instead of `c.ID()`.